app/views/static_pages/index.html.erb

```diff
@@ -1,2 +1,109 @@
-<h1>Home Page</h1>
+<html>
+  <body>
+    <br />
```

Comment:
So, `<br />` tags are used for line breaks inside a paragraph, or when you need to move to the next line without starting a new paragraph.  They are not necessarily supposed to be used for formatting, as in you'd use them when the paragraph needed a line break, not when you want blank space.  For blank space, there's the css `margin` and `padding`.

Anyways, it's probably okay to leave them here for now, unless you want to remove them, but by the end we should move all the spacing over to CSS rules.